docker/README.md

```diff
@@ -0,0 +1,79 @@
+# Using Docker to run garage
+
+Currently there are three types of garage images:
+  - ci: used for the continuous integration of garage in TravisCI and not
+    intended for final users.
+  - headless: garage without environment visualization.
+  - NVIDIA: garage with environment visualization using an NVIDIA graphics
+    card.
+
```

Comment:
what about users who want to run interactively but without an nvidia card? see docker-opengl

Reply:
I've checked docker-opengl, but it doesn't work when installing some apt dependencies. I'm still working on the opengl flavor.

Reply:
okay. can you explain the difference between headless and CI then? i am still confused